### DISCUSSION TITLE
You dropped by today, leave your thoughts!

### DISCUSSION THREAD

**Discussion by crowbarmaster** [crowbarmaster/BedrockManagementService#1] [Ideas]
Hello everyone! Hard at work refactoring/restructuring the source now. Project grew fairly large quickly and it became a hastle to work in. Pretty close to a new release! I would like to hear from you! Your thoughts about what has been done, what features you think could be useful or even your own modifications are all welcome! Your time and interest are appreciated!

-Crowbarmaster.

**Comment by ElectronJunkie** [comment]
Stumbled across your project after a long break from running a Minecraft server and I must say this is one of the best GUI's I've seen! You've managed to cover all the bases in terms of features and other than a few lags on the interface (I'm RDP'd so it could just be that) it works smooth as butter! Keep up the great work!

**Reply by crowbarmaster** [reply]
Hello! Thanks for the kind thoughts! This has had quite a few evolutions to get here now, and I have a few friends here to thank and add to the credits when I get some time to wrap things up with the readme. I had more time up until recently, so progress has been a bit slow. With that in mind, I do believe adding new features will probably come to a trickle. Hopefully I can come up with some steady time to keep at it soon!! Thanks again!
